cmake/templates/vpConfig.h.in

```diff
@@ -480,6 +480,15 @@
 // Defined if Point Cloud Library is available
 #cmakedefine VISP_HAVE_PCL
 
+// Defined if required PCL components are found
+#if defined(VISP_HAVE_PCL)
+#cmakedefine VISP_HAVE_PCL_COMMON
+#cmakedefine VISP_HAVE_PCL_FILTERS
+#cmakedefine VISP_HAVE_PCL_IO
```

Comment:
In your PR I don't see the usage of
- `VISP_HAVE_PCL_FILTERS`
- `VISP_HAVE_PCL_SEGMENTATION`

Reply:
Forgot to check them in vpMbtFaceDepthNormal. done in 3a25ffff3